docker/actinia-stac-tests/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM mundialis/actinia-core:2.0.0 as actinia_test
+
+LABEL authors="Carmen Tawalika,Anika Weinmann"
+LABEL maintainer="tawalika@mundialis.de,weinmann@mundialis.de"
+
+ENV SETUPTOOLS_SCM_PRETEND_VERSION=0.0.0
+ENV ACTINIA_CUSTOM_TEST_CFG /etc/default/actinia_stac_test
+
+# TODO do not set DEFAULT_CONFIG_PATH if this is fixed
+ENV DEFAULT_CONFIG_PATH /etc/default/actinia_stac_test
+
+# install things only for tests
+RUN apk add redis
+RUN pip3 install iniconfig colorlog
+
+# uninstall actinia core from FROM-image
+#RUN pip3 uninstall actinia-core -y
+
+# copy needed files and configs for test
+COPY docker/actinia-stac-tests/actinia-stac-test.cfg /etc/default/actinia_stac_test
+COPY . /src/actinia-stac-plugin/
```

Comment:
It makes sense to do this step as late as possible to use as much build cache as possible because this COPY will break the cache for any changes in the repo in files not in `.dockerignore' file. Therefore it can be moved just before the `make install` command.